src/main/java/se/sundsvall/supportmanagement/api/model/communication/CommunicationAttachment.java

```diff
@@ -0,0 +1,98 @@
+package se.sundsvall.supportmanagement.api.model.communication;
+
+import static io.swagger.v3.oas.annotations.media.Schema.RequiredMode.REQUIRED;
+
+import java.util.Objects;
+
+import jakarta.validation.constraints.NotBlank;
+
+import io.swagger.v3.oas.annotations.media.Schema;
+
+public class CommunicationAttachment {
+
+	@NotBlank
+	@Schema(
+		description = "The attachment ID",
+		example = "aGVsbG8gd29ybGQK",
+		requiredMode = REQUIRED
```

Comment:
Is requredMode needed as the class is only used in response from service (comment applicable for all places where requiredMode is defined in class)?